mmv1/templates/terraform/examples/datastream_stream_sql_server.tf.erb

```diff
@@ -83,6 +83,7 @@ resource "google_datastream_stream" "default" {
                     }
                 }
             }
+            change_tables {}
```

Comment:
we'll want to also include a test that uses the `transcation_logs` field also.

Reply:
I refactored this test to be for the `transaction_logs` (as that is the default) and added a test for the `change_tables` in  this commit: a06af09fb62ca6f4d5a8749485dc5d65e390d95f

I set `skip_test: true` based on a [previous conversation](https://github.com/GoogleCloudPlatform/magic-modules/pull/10553#issuecomment-2091882863) in another one of my datastream PRs. To run the tests would require configuration on the SQL Server instance. I noted this in the `Stream.yaml` as well.

Reply:
tests are looking good, I'll run the test on my machine before approving!

Reply:
Let me know if you have a chance to run the tests. I can provide the debug logs from my local if it helps.

Reply:
Any updates @BBBmau

Reply:
Waiting for this work by @nbugden PR to be merged as well @BBBmau

Reply:
@BBBmau any updates? Would like to wrap this up.

Reply:
@nbugden  apologies for the delay, only thing left for me is to manual test this. I'll be sure to do this before the end of the week.

Reply:
If it helps I can provide the debug logs from my local. Manual testing will require you configure CDC on the table for testing the `change_tables`

Reply:
This would be helpful, providing a simple tfconfig for reproduction would also help as well!

Reply:
Adding @yuval-badal to the convo as he provided the DBA expertise for sqlserver on my end.

@BBBmau Attached is:
* example.tf - with example config for setting up the stream
* change_tables.output - tf debug logs when creating a stream using the new `change_tables` block
* transaction_logs.output - tf debug logs when updating the stream using the new `transaction_logs` block
* original.output - tf debug logs when updating the stream by removing the `transactio_logs` block

A few things to note:
* I didn't include config for the sql server instance. This is out of scope what the PR since SQL server config would be the repsonsiblity of the DBA. That said, for this to work it requires you have a SQL Server instance set up with:
  * CDC enabled on the db and the table
    Enable CDC on the DB
    ```sql
    USE db;
    EXEC sys.sp_cdc_enable_db;
    ```
    Enable CDC on the Table
    ```sql
    EXEC sys.sp_cdc_enable_table
    @source_schema = N'test',
    @source_name = N'test_table',
    @supports_net_changes = 0,
    @role_name = NULL; 
    ```
  * By default you can set up streams for change_tables. To test the transaction_logs you must update the polling interval.
    ```sql
    EXEC sys.sp_cdc_change_job @job_type = 'capture' , @pollinginterval = 86399
    EXEC sp_cdc_stop_job 'capture'. 
    EXEC sp_cdc_start_job 'capture'    
    ```

[example.tf.txt](https://github.com/user-attachments/files/16599763/example.tf.txt)
[change_tables.output.txt](https://github.com/user-attachments/files/16600312/change_tables.output.txt)
[transaction_logs.output.txt](https://github.com/user-attachments/files/16600422/transaction_logs.output.txt)
[original.output.txt](https://github.com/user-attachments/files/16600427/original.output.txt)

Reply:
@BBBmau have you had a chance to review?

Reply:
@BBBmau any updates?